### ISSUE TITLE
Add nl-core-Patient to most nl-core- profiles

### ISSUE THREAD

**Issue by pieter-edelman-nictiz** [Nictiz/Nictiz-R4-zib2020#183]
For profiles where `.subject` is required, we usually add the nl-core-Patient profile as target. However, zibs are implicitly always about a patient (unless they aren't). Should we make sure the nl-core-Patient is (almost) always added to nl-core- profiles? And should we try to add this to the automated checks?

**Comment by ArdonToonstra** [comment]
Jorn will perform some XPath magic to analyse the situation.

**Comment by jduwel** [comment]
Added one missing targetProfile in LivingSituation on branch 183 linked to the right.

Used the following Xpath to locate all SDs where no nl-core-Patient targetProfile was missing. 
`/f:StructureDefinition[not(f:differential/f:element/f:type/f:targetProfile[contains(@value, 'nl-core-Patient')])]`
Aside from the usual suspects, LivingSituation was the only profile where it was missing.

Up for debate in my opinion is nl-core-MedicalDevice.Product (https://simplifier.net/nictiz-r4-zib2020/nlcoremedicaldeviceproduct). ResourceType Device has a `.patient` element, but this is more then the subject of the (use of?) the Device, its definition speaks of 'affixed to a patient'. I did not add nl-core-Patient here.
